src/Classes/Utils/CanvasUtils.ts

```diff
@@ -23,49 +22,57 @@ export class CanvasUtils {
     }
 
     public static drawPolygonMask(context: CanvasRenderingContext2D,
-                                  rawData: ICoordinates[],
-                                  stroke: IPolygonMaskDrawStroke): void {
-        context.save();
-        context.beginPath();
-        context.moveTo(rawData[0].x, rawData[0].y);
+        rawData: ICoordinates[],
+        stroke: IPolygonMaskDrawStroke): void {
+        const color = typeof stroke.color === "string" ? ColorUtils.stringToRgb(stroke.color) : ColorUtils.colorToRgb(stroke.color);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)